### ISSUE TITLE
Eval statements in Setup your Mac specify executable twice

### ISSUE THREAD

**Issue by pyther** [dan-snelson/dialog-scripts#3]
The eval statements in the `Setup your Mac.bash` script list /usr/local/bin/dialog twice.

Code
```
eval "$dialogApp" "${dialogCMD[*]}" & sleep 0.3
```

PS Output
```
/bin/sh /usr/local/bin/dialog /usr/local/bin/dialog --ontop --title Setting up your Mac --message Please wait while the following apps are installed … --icon SF=laptopcomputer.and.arrow.down,weight=semibold,colour1=#ef9d51,colour2=#ef7951 --progress --button1text Quit --button1disabled --infotext v1.2.4 --titlefont size=28 --messagefont size=14 --height 67% --position centre --blurscreen --overlayicon /Applications/Self Service.app --quitkey K
```

Potential fix
```
eval $dialogCMD & sleep 0.3
```

**Comment by dan-snelson** [comment]
Thanks for the heads-up, @pyther!

Both the following lines …
- https://github.com/dan-snelson/dialog-scripts/blob/main/Setup%20Your%20Mac/Setup-Your-Mac-via-Dialog.bash#L198
- https://github.com/dan-snelson/dialog-scripts/blob/main/Setup%20Your%20Mac/Setup-Your-Mac-via-Dialog.bash#L237

… already include `$dialogApp`.

So, the following lines …
- https://github.com/dan-snelson/dialog-scripts/blob/main/Setup%20Your%20Mac/Setup-Your-Mac-via-Dialog.bash#L464
- https://github.com/dan-snelson/dialog-scripts/blob/main/Setup%20Your%20Mac/Setup-Your-Mac-via-Dialog.bash#L484

… should be like:
- https://github.com/dan-snelson/dialog-scripts/blob/main/Setup%20Your%20Mac/Setup-Your-Mac-via-Dialog.bash#L442

**Comment by dan-snelson** [comment]
@pyther Please test [v1.2.5 from the dev branch](https://github.com/dan-snelson/dialog-scripts/blob/development/Setup%20Your%20Mac/Setup-Your-Mac-via-Dialog.bash).

**Comment by dan-snelson** [comment]
Resolved in `1.2.5`.
